test/specs/wrapper/setValue.spec.js

```diff
@@ -0,0 +1,69 @@
+import ComponentWithInput from '~resources/components/component-with-input.vue'
+import { describeWithShallowAndMount } from '~resources/utils'
+
+describeWithShallowAndMount('setValue', (mountingMethod) => {
+  it('sets element value', () => {
+    const wrapper = mountingMethod(ComponentWithInput)
+    const input = wrapper.find('input[type="text"]')
+    input.setValue('foo')
+
+    expect(input.element.value).to.equal('foo')
+  })
+
+  it('calls trigger with input event', () => {
```

Comment:
We can remove this test, as long as the dom updates with the v-model, it doesn't matter whether we call trigger or dispatch an event directly.